src/wp-admin/includes/upgrade.php

```diff
@@ -464,18 +418,18 @@ function wp_install_defaults( $user_id ) {
 			array(
 				'wp_inactive_widgets' => array(),
 				'sidebar-1'           => array(
-					0 => 'search-2',
-					1 => 'recent-posts-2',
-					2 => 'recent-comments-2',
+					0 => 'block-2',
+					1 => 'block-3',
+					2 => 'block-4',
 				),
 				'sidebar-2'           => array(
-					0 => 'archives-2',
-					1 => 'categories-2',
-					2 => 'meta-2',
+					0 => 'block-5',
+					1 => 'block-6',
```

Comment:
Shouldn't we still have a meta widget in here?

Reply:
I can't decide... would it be weird to have a Legacy Widget block in amongst the default blocks that you see in a new WordPress site? Is it a useful widget to have by default in 2021?

Reply:
Yeah good point about the Legacy Widget. We can leave it out for now and see if anyone notices 😄

Reply:
I think it makes sense to not have this legacy widget on a new install. Why? The default experience of a new install does not include the Meta widget in an active sidebar.

That said, for those who want to use its features, might be good to have a new block version of it.

Reply:
That makes sense to me. I've kept it removed.

Reply:
My worry here is that we assume it's not useful. Isn't the meta widget the default front end UX path to wp-admin?

Reply:
I would vote for leaving the Meta widget out but would it be possible for it to be available in the inserter and in the Legacy Widget block dropdown list until a new block version is created? (Similar to what we’re looking at doing for the Navigation Menu widget in [#32431](https://github.com/WordPress/gutenberg/pull/32431)?)

Reply:
> My worry here is that we assume it's not useful. Isn't the meta widget the default front end UX path to wp-admin?

It already does not show on a new WordPress website because Twenty Twenty-one only has one widget area.

> I would vote for leaving the Meta widget out but would it be possible for it to be available in the inserter and in the Legacy Widget block dropdown list until a new block version is created? (Similar to what we’re looking at doing for the Navigation Menu widget in #32431?)

Yes absolutely. Would you mind creating an issue in the Gutenberg repo about adding a block equivalent of Meta? I looked but couldn't find one.

Reply:
No prob, made an issue for the block version here: https://github.com/WordPress/gutenberg/issues/32551